src/Appwrite/Platform/Workers/Messaging.php

```diff
@@ -409,7 +409,8 @@ private function sendInternalSMSMessage(Document $message, Document $project, ar
                 ],
                 'msg91' => [
                     'senderId' => $user,
-                    'authKey' => $password
+                    'authKey' => $password,
+                    'templateId' => $smsDSN->getParam('templateId'),
```

Comment:
Previous versions used the `_APP_SMS_FROM` env var so we should fallback to that.

```suggestion
                    'templateId' => $smsDSN->getParam('templateId', $from),
```

Reply:
Okay lets apply this suggestion 👍